common/adminBlock/EntryServerFault.go

```diff
@@ -10,10 +10,12 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
+// ServerFault is an admin block entry which contains all the information and messages from an election, which includes the removal and
```

Comment:
Just for your knowledge base, this is no longer used, but might be revisted and used again

Reply:
added a comment as such